### ISSUE TITLE
Bug in DatePicker.Component.FullDay

### ISSUE THREAD

**Issue by bardu** [IjzerenHein/famous-flex#125]
I couldn't figure out the source of this bug in the source code, however, DatePicker.Component.FullDay returns the correct date only in January. If you spin the component, for example to Tu 02 Feb new Date(datePicker.getDate()) will return 2016-01-02... .So, the month is always returned as 01 (January)

You can test this properly with your one examples. 

My DatePicker looks like this:
```js
this.datePicker.setComponents([
            new DatePicker.Component.FullDay({
                format: function(date) {
                    return moment(date).format('dd DD MMM');
                },
                sizeRatio: 2
            }),
            new DatePicker.Component.Hour(),
            new DatePicker.Component.Minute()
        ]);
```

**Comment by IjzerenHein** [comment]
Ah, great find! I'll have a look at this very soon.
cheers

**Comment by bardu** [comment]
So, now we have February and DatePicker.Component.FullDay returns the correct date, however, when I spin the component to a date in March it returns the February month.

Hope that helps.

**Comment by IjzerenHein** [comment]
Hi, I've fixed the issue with the DatePicker.Component.FullDay component. The changes are on the develop branch. Two other issues have also been addresses. There was a bug which caused the date-picker to not show the initial date correctly, and it's now possible to click on a date/time item in order to select it. I will release an official bugfix version shortly.
cheers

**Comment by bardu** [comment]
> I will release an official bugfix version shortly.

@IjzerenHein  Any date for that?

**Comment by IjzerenHein** [comment]
Good thing you asked. I'll be doing that this week!

**Comment by IjzerenHein** [comment]
Hi, I've released an update v0.3.7. It addresses this and several other issues with the DatePicker widget.
cheers

**Comment by bardu** [comment]
Thanks Hein, much appreciated.
